README.md

```diff
@@ -22,8 +22,8 @@ $ helm repo add deis https://github.com/deis/charts
 $ helm up
 $ helm fetch deis/deis
 $ helm install deis
```

Comment:
yeah but it's missing on this line as well. The official instructions say to use `helm install deis --namespace=deis` but it's not reflected here, just on the `kubectl` commands below. :)